src/main/java/com/jfrog/ide/idea/ui/issues/ComponentIssueDetails.java

```diff
@@ -17,8 +17,6 @@ private ComponentIssueDetails(DependenciesTree node) {
         super(node);
         Issue topIssue = node.getTopIssue();
         addText("Top Issue Severity:", StringUtils.capitalize(topIssue.getSeverity().toString()));
-        addText("Top Issue Type:", StringUtils.capitalize(topIssue.getIssueType()));
-        addText("Issues Count:", String.valueOf(node.getIssueCount()));
```

Comment:
Please don't delete the issues count.

Reply:
@eyalbe4 we talked about removing the 'Issues Count' property, what do you think?

Reply:
OK I talked with him. We agreed to remove the issues count.